File: multi_language_code_generator/implementations/go/order_processor.go
package main

import "fmt"

type OrderResponse struct {
    Item   string  json:"item"
    Total  float64 json:"total"
    Status string  json:"status"
}

func ProcessOrder(item string, price float64, quantity int) OrderResponse {
    total := price * float64(quantity)
    status := "Invalid"
    if total > 0 {
        status = "Processed"
    }
    return OrderResponse{Item: item, Total: total, Status: status}
}

func main() {
    fmt.Printf("%+v\n", ProcessOrder("Laptop", 1200, 2))
}

File: multi_language_code_generator/implementations/javascript/order_processor.js
function processOrder(item, price, quantity) {
    const total = price * quantity;
    const status = total > 0 ? "Processed" : "Invalid";
    return {
        item: item,
        total: total,
        status: status
    };
}

console.log(processOrder("Laptop", 1200, 2));
